wpeverest/user-registration
# Tweak - Default value for country field

### All Submissions:


### Changes proposed in this Pull Request:

Previously, there was no value for the default value list in the advanced setting of the country field after dragging the country field. This PR fixes the list of default values. And also this PR enhances Confirm password and Email as default required, User can't change it from the field option tab.

### How to test the changes in this Pull Request:

1. Create or edit a form -> drag a country field -> click field options -> click on advanced setting-> check the default values
2.  And drag confirm email and confirm password field -> click field options -> check whether we can change the required or not.
3. Verify the validation message of confirm email and confirm password field by submitting or changing the value

### Types of changes:


### Other information:


### Changelog entry

> Tweak - Default value for country field
